_layouts/default.html

```diff
@@ -141,8 +141,8 @@
             });
         </script>
     {% endif %}
-
-    {% include schema-event.html %}
+    
+    <!-- {% include schema-event.html %} -->
```

Comment:
As discussed, JSON-LD schema(structured data) should be inside the head tag. So we will keep those markups in head.html